File: output/csharp/Cat/CatHealth/CatHealthRecord.cs
using Nest.Cat;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cat {

	public class CatHealthRecord : ICatRecord {
		
		[DataMember(Name="cluster")]
		public string Cluster { get; set; }


		[DataMember(Name="epoch")]
		public string Epoch { get; set; }


		[DataMember(Name="init")]
		public string Init { get; set; }


		[DataMember(Name="node.data")]
		public string NodeData { get; set; }


		[DataMember(Name="node.total")]
		public string NodeTotal { get; set; }


		[DataMember(Name="pending_tasks")]
		public string PendingTasks { get; set; }


		[DataMember(Name="pri")]
		public string Pri { get; set; }


		[DataMember(Name="relo")]
		public string Relo { get; set; }


		[DataMember(Name="shards")]
		public string Shards { get; set; }


		[DataMember(Name="status")]
		public string Status { get; set; }


		[DataMember(Name="timestamp")]
		public string Timestamp { get; set; }


		[DataMember(Name="unassign")]
		public string Unassign { get; set; }

	}
}

File: specification/specs/cat/cat_health/CatHealthRecord.ts
/*
 * Licensed to Elasticsearch B.V. under one or more contributor
 * license agreements. See the NOTICE file distributed with
 * this work for additional information regarding copyright
 * ownership. Elasticsearch B.V. licenses this file to you under
 * the Apache License, Version 2.0 (the "License"); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */

class CatHealthRecord {
  cluster: string
  epoch?: string
  init: string
  'node.data': string
  'node.total': string
  pending_tasks: string
  pri: string
  relo: string
  shards: string
  status: string
  timestamp?: string
  unassign: string
  max_task_wait_time: string
  active_shards_percent: string
}
